packages/ui/components/pages/Fuse/Modals/PoolModal/AmountSelect.tsx

```diff
@@ -307,45 +289,84 @@ const AmountSelect = ({
             width="100%"
           >
             <Column mainAxisAlignment="flex-start" crossAxisAlignment="flex-start" width="100%">
+              {/* Operation Selection */}
               <TabBar
                 color="black"
                 mode={mode}
                 setMode={setMode}
                 setUserEnteredAmount={_setUserEnteredAmount}
                 setAmount={_setAmount}
               />
+
+              {/* Asset Balance */}
               <Row width="100%" mt={4} mainAxisAlignment="flex-end" crossAxisAlignment="center">
                 <Text mr={2}>Wallet Balance:</Text>
                 <Text>
                   {myBalance ? utils.formatUnits(myBalance, asset.underlyingDecimals) : 0}{' '}
                   {asset.underlyingSymbol}
                 </Text>
               </Row>
-              <DashboardBox width="100%" height="70px" mt={3}>
-                <Row
-                  width="100%"
-                  p={4}
-                  mainAxisAlignment="space-between"
-                  crossAxisAlignment="center"
-                  expand
-                >
-                  <AmountInput
-                    displayAmount={userEnteredAmount}
-                    updateAmount={updateAmount}
-                    disabled={isBorrowPaused}
-                    autoFocus
-                  />
-                  <TokenNameAndMaxButton mode={mode} asset={asset} updateAmount={updateAmount} />
-                </Row>
-              </DashboardBox>
-              {mode === FundOperationMode.BORROW && borrowableAmount + borrowedAmount !== 0 && (
-                <MaxBorrowSlider
-                  userEnteredAmount={userEnteredAmount}
-                  updateAmount={updateAmount}
-                  borrowableAmount={borrowableAmount}
-                  borrowedAmount={borrowedAmount}
-                  underlyingPrice={asset.underlyingPrice}
-                />
+
+              {mode === FundOperationMode.BORROW && asset.liquidity.isZero() ? (
+                <Alert status="info">
+                  <AlertIcon />
+                  Unable to borrow this asset yet. The asset does not have enough liquidity.
+                  <br /> Feel free to supply this asset to be borrowed by others in this pool to
+                  earn interest.
+                </Alert>
+              ) : (
+                <>
+                  {mode === FundOperationMode.BORROW && (
+                    <Row
+                      width="100%"
+                      mt={4}
+                      mainAxisAlignment="flex-end"
+                      crossAxisAlignment="center"
+                    >
+                      <Alert status="info" pb={4}>
+                        <AlertIcon />
+                        {`For safety reasons, you need to borrow at least a value of $${
+                          minBorrowUSD ? minBorrowUSD?.toFixed(2) : 100
```

Comment:
Should we set this in the SDK side as a config parameters? Not necessarily in this PR

Reply:
Ah, sorry! Yes, that's perfect.